pom.xml

```diff
@@ -67,9 +67,9 @@
     <app.parents>system:cdap-data-pipeline[4.0.0,7.0.0),system:cdap-data-streams[4.0.0,7.0.0)</app.parents>
     <!-- this is here because project.basedir evaluates to null in the script build step -->
     <main.basedir>${project.basedir}</main.basedir>
-    <cdap.version>6.1.0-SNAPSHOT</cdap.version>
+    <cdap.version>6.1.1</cdap.version>
```

Comment:
Set it to 6.1.2 since hydrator-plugins 2.3.5 depends on cdap 6.1.2.
https://github.com/cdapio/hydrator-plugins/blob/v2.3.5/pom.xml#L681

Reply:
Done